### PR TITLE
Add-AzsScaleUnitNode + CI bugfixing

### PR DESCRIPTION
**Pull request by LingyunSu**
1. Add-AzsScaleUnitNode 
2. CI bugfixing
3. Remove comments in custom files to fix missing synopsis

### CODE REVIEW

File: src/Azs.Fabric.Admin/custom/Add-AzsScaleUnitNode.ps1 [hunk 1]
```diff
@@ -0,0 +1,191 @@
...
+    ${InputObject},
+
+    [Parameter(ParameterSetName='ScaleExpanded')]
+    [Parameter(ParameterSetName='ScaleViaIdentityExpanded')]
+    [Microsoft.Azure.PowerShell.Cmdlets.FabricAdmin.Category('Body')]
+    [System.Management.Automation.SwitchParameter]
+    # Flag indicates if the operation should wait for storage to converge before returning.
+    ${AwaitStorageConvergence},
+
+	[Parameter(ParameterSetName='BmciPv4Address', Mandatory)]
```
Comment (bganapa) [inline src/Azs.Fabric.Admin/custom/Add-AzsScaleUnitNode.ps1:94]: this is a new parameter set?

File: src/Azs.Fabric.Admin/custom/Add-AzsScaleUnitNode.ps1 [hunk 2]
```diff
@@ -0,0 +1,191 @@
...
+    [Parameter(ParameterSetName='ScaleExpanded', Mandatory)]
+    [Microsoft.Azure.PowerShell.Cmdlets.FabricAdmin.Category('Path')]
+    [System.String]
+    # BMC address of the physical machine.
+    ${BmciPv4Address},
+
+	[Parameter(ParameterSetName='ComputerName ', Mandatory)]
```
Comment (bganapa) [inline src/Azs.Fabric.Admin/custom/Add-AzsScaleUnitNode.ps1:101]: nit : extra white spaces in fornt

File: src/Azs.Fabric.Admin/custom/Add-AzsScaleUnitNode.ps1 [hunk 3]
```diff
@@ -0,0 +1,191 @@
...
+    [Parameter(ParameterSetName='ScaleExpanded', Mandatory)]
```
Comment (bganapa) [inline src/Azs.Fabric.Admin/custom/Add-AzsScaleUnitNode.ps1:102]: ComputerName  and BmciPv4Address parameters are in two different parameter set, i think you want them to be mandatory in the same parameter set

File: src/Azs.Fabric.Admin/custom/Add-AzsScaleUnitNode.ps1 [hunk 4]
```diff
@@ -0,0 +1,191 @@
...
+    [Microsoft.Azure.PowerShell.Cmdlets.FabricAdmin.Category('Path')]
+    [System.String]
+    # Computer name of the physical machine.
```
Comment (bganapa) [inline src/Azs.Fabric.Admin/custom/Add-AzsScaleUnitNode.ps1:105]: recommend adding [ValidateNotNull()]

File: src/Azs.Fabric.Admin/custom/Add-AzsScaleUnitNode.ps1 [hunk 5]
```diff
@@ -0,0 +1,191 @@
...
+		
+		if ($PSBoundParameters.ContainsKey(('BmciPv4Address'))){
+            $null = $PSBoundParameters.Remove('BmciPv4Address')
+        }
+				
+		if ($PSBoundParameters.ContainsKey(('ComputerName'))){
+            $null = $PSBoundParameters.Remove('ComputerName')
+        }	
+
+		Azs.Fabric.Admin.internal\Add-AzsScaleUnitNode @PSBoundParameters
```
Comment (bganapa) [inline src/Azs.Fabric.Admin/custom/Add-AzsScaleUnitNode.ps1:189]: I like the way you have customized the cmdlet here..

File: src/Azs.Fabric.Admin/custom/Add-AzsScaleUnitNode.ps1 (hunk 1 above)
Comment (LingyunSu) [reply src/Azs.Fabric.Admin/custom/Add-AzsScaleUnitNode.ps1:94]: Is is redundant.

File: src/Azs.Fabric.Admin/custom/Add-AzsScaleUnitNode.ps1 (hunk 3 above)
Comment (LingyunSu) [reply src/Azs.Fabric.Admin/custom/Add-AzsScaleUnitNode.ps1:102]: "ScaleExpanded" is the right parameter set for both of them, good catch!

**Pull request merged**